Makefile

```diff
@@ -79,9 +88,22 @@ release : docker
 	@docker login -e $(DOCKER_EMAIL) -u $(DOCKER_USERNAME) -p $(DOCKER_PASSWORD)
 	@for build in $(builds) ; do \
 	  set -e; \
-	  tag=skycirrus/feed-$$build:$(git_rev) ; \
-	  latest_tag=skycirrus/feed-$$build:latest ; \
+	  tag=${repository}/feed-$$build:${build_version} ; \
+	  latest_tag=${repository}/feed-$$build:latest ; \
 	  docker tag $$tag $$latest_tag ; \
 	  docker push $$tag ; \
 	  docker push $$latest_tag ; \
 	done
+
+releasetotest : docker
+	@echo "== release docker image to the test repository"
+	@for build in $(builds) ; do \
+	  set -e; \
+	  image_tag=${repository}/feed-$$build:${build_version} ; \
+	  test_tag=${repository}/test/feed-$$build:${build_version} ; \
+	  latest_tag=${repository}/test/feed-$$build:latest ; \
+	  docker tag $$image_tag $$test_tag ; \
+	  docker tag $$test_tag $$latest_tag ; \
+	  docker push $$test_tag ; \
+	  docker push $$latest_tag ; \
+	done
```

Comment:
We need a test repository that does not require login in - the problem with dockerhub is that you need to login in order to push any image.  
Besides you have to tag the image with the correct repo name before it can be pushed it, and of course before you do that you need to build it. That's 3 commands. So `releasetotest`  is simply a mechanism to do all that in one command without having to log in.